frontend/htmlTemplate.ts

```diff
@@ -61,14 +61,48 @@ export default ({
             <body>
                 <div id='app'>${html}</div>
                 <script>
-                window.gu = ${sanitiseDomRefs(
+                window.guardian = ${sanitiseDomRefs(
                     JSON.stringify({
                         app: {
                             data,
                             cssIDs,
                         },
+                        config: {
+                            ophan: {
+                                // This is duplicated from
+                                // https://github.com/guardian/ophan/blob/master/tracker-js/assets/coffee/ophan/transmit.coffee
+                                // Please do not change this without talking to the Ophan project first.
+                                // WHY?
+                                // https://github.com/guardian/frontend/pull/9982
+                                pageViewId:
```

Comment:
We should be generating this `pageViewId` in the client, currently this will run on the server and pageViewId will be in the server side rendered page?